vars/piperStageWrapper.groovy

```diff
@@ -95,7 +95,7 @@ private void executeStage(script, originalStage, stageName, config, utils, telem
         }
 
         // Second, check if a project extension (within the same repository) exists
-        if ((env.PIPER_DISABLE_EXTENSIONS ==null || projectExtensions) && (!Boolean.valueOf(script.env.PIPER_DISABLE_EXTENSIONS) && projectExtensions)) {
+        if ((script.env.PIPER_DISABLE_EXTENSIONS==null || projectExtensions) && (!Boolean.valueOf(script.env.PIPER_DISABLE_EXTENSIONS) && projectExtensions)) {
```

Comment:
This condition check in fact works, Is there any issue in defining it like this?

Reply:
No, you can revert the change. I just try to figure out why the test fails.

Reply:
i just wan't to inform about an issue we have wrt tests that depends on each other. Don't know if that relates here ... See [here](https://github.com/SAP/jenkins-library/issues/1313#issuecomment-605838779)

Reply:
@srinikitha09 now the tests are green, so feel free to revert to the previous state. Just make sure not to use `script.env` but simply `env`.

Reply:
Code looks cleaner now. I do not want to revert to  the previous state.